crates/qt-build-utils/src/lib.rs

```diff
@@ -11,6 +11,8 @@
 //! [cxx_build](https://docs.rs/cxx-build/latest/cxx_build/), or
 //! [cpp_build](https://docs.rs/cpp_build/latest/cpp_build/).
 
+#![allow(clippy::too_many_arguments)]
```

Comment:
assume this is for the `cargo_link_qt_library` ? can this be put just on the method rather than enabled for the whole module/crate ?

Reply:
Ah or is this for multiple things i see register_qml_module and find_qt_module_prl as well

Reply:
Maybe points to needing to pass around a struct or something later to avoid the many arguments.

Reply:
> Maybe points to needing to pass around a struct or something later to avoid the many arguments.

Yes, that's the point of the clippy warning. But I didn't want it to get too confusing with 3 different QmlModule structs :upside_down_face:

Reply:
Annoyingly this has to be done on the module level; it doesn't work on functions.